app/src/main/java/org/inspire/breath/WawReader.java

```diff
@@ -0,0 +1,70 @@
+package org.inspire.breath;
+import java.io.*;
+
+public class WawReader {
+
+    public static void main(String[] args)
```

Comment:
This main method doesn't do anything and won't run in the app, probably better to remove it